CONTRIBUTING.md

```diff
@@ -1,172 +1,76 @@
+2. Determine whether a GitHub issue or discussion is more appropriate for your needs:
+   1. If want to propose something new that requires specification or an additional design, or you would like to change a process, start with a [new discussion](https://github.com/cosmos/gaia/discussions/new/choose). With discussions, we can better handle the design process using discussion threads. A discussion usually leads to one or more issues.
+   2. If the issue you want addressed is a specific proposal or a bug, then open a [new issue](https://github.com/cosmos/gaia/issues/new/choose).
+   3. Review existing [issues](https://github.com/cosmos/gaia/issues) to find an issue you'd like to help with.
+3. Participate in thoughtful discussion on that issue.
+4. If you would like to contribute:
+   1. Ensure that the proposal has been accepted.
+   2. Ensure that nobody else has already begun working on this issue. If they have,
+      make sure to contact them to collaborate.
+   3. If nobody has been assigned for the issue and you would like to work on it,
+      make a comment on the issue to inform the community of your intentions
+      to begin work.
+5. To submit your work as a contribution to the repository follow standard GitHub best practices. See [pull request guideline](#pull-requests) below.
+
+**Note:** For very small or blatantly obvious problems such as typos, you are
 not required to an open issue to submit a PR, but be aware that for more complex
 problems/features, if a PR is opened before an adequate design discussion has
-taken place in a github issue, that PR runs a high likelihood of being rejected.
-
-Take a peek at our [coding repo](https://github.com/tendermint/coding) for
-overall information on repository workflow and standards. Note, we use `make
-tools` for installing the linting tools.
-
-Other notes:
-
-- Looking for a good place to start contributing? How about checking out some
-  [good first issues](https://github.com/cosmos/gaia/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-- Please make sure to run `make format` before every commit - the easiest way
-  to do this is have your editor run it for you upon saving a file. Additionally
-  please ensure that your code is lint compliant by running `golangci-lint run`.
-  A convenience git `pre-commit` hook that runs the formatters automatically
-  before each commit is available in the `contrib/githooks/` directory.
-
-## Submissions
+taken place in a GitHub issue, that PR runs a high likelihood of being rejected 
+without review.
 
-Generally, when structuring a submission for the Cosmos ecosystem, the submission might span multiple layers
-of the technology stack, including the Cosmos SDK and Tendermint dependencies.
+## Teams Dev Calls
```

Comment:
This section doesn't talk about Team Dev Calls as far as I see.
Maybe rename it? Alternatively (or in addition) add info about dev calls if we intend to have them

```suggestion
## Team Organization
```